web/src/components/users/FirstUserForm.tsx

```diff
@@ -207,99 +199,75 @@ export default function FirstUserForm() {
-                      setInsideDropDown={setInsideDropDown}
-                      focusedIndex={focusedIndex}
-                    />
-                  </FormGroup>
-                </Stack>
-              </Page.Section>
-            </GridItem>
-            <GridItem sm={12} xl={6}>
-              <Page.Section>
-                <Stack hasGutter>
-                  {state.isEditing && (
-                    <Switch
-                      label={_("Edit password too")}
-                      isChecked={changePassword}
-                      onChange={() => setChangePassword(!changePassword)}
-                    />
-                  )}
-                  <PasswordAndConfirmationInput
-                    inputRef={passwordRef}
-                    isDisabled={!changePassword}
-                    showErrors={false}
-                  />
-                </Stack>
-              </Page.Section>
-            </GridItem>
-            <GridItem sm={12} xl={6}>
-              <Page.Section>
-                <Checkbox
-                  aria-label={_("user autologin")}
-                  id="autologin"
-                  name="autologin"
-                  // TRANSLATORS: check box label
-                  label={_("Auto-login")}
-                  defaultChecked={state.user.autologin}
-                />
-              </Page.Section>
-            </GridItem>
-          </Grid>
+          <FormGroup fieldId="userFullName" label={_("Full name")}>
+            <TextInput
+              id="userFullName"
+              name="fullName"
+              value={fullName}
+              onChange={(_, value) => setFullName(value)}
+              onBlur={(e) => setSuggestions(suggestUsernames(e.target.value))}
+            />
+          </FormGroup>
+          <FormGroup className="first-username-wrapper" fieldId="userName" label={_("Username")}>
+            <TextInput
+              id="userName"
+              name="userName"
+              ref={usernameInputRef}
+              value={userName}
+              isRequired
+              onChange={(_, value) => setUserName(value)}
+              onFocus={renderSuggestions}
+              onKeyDown={handleKeyDown}
+              onBlur={() => !insideDropDown && setShowSuggestions(false)}
+            />
+            <UsernameSuggestions
+              isOpen={showSuggestions}
+              entries={suggestions}
+              onSelect={onSuggestionSelected}
+              setInsideDropDown={setInsideDropDown}
+              focusedIndex={focusedIndex}
+            />
+          </FormGroup>
+          {isEditing && (
+            <Switch
+              label={_("Edit the password too")}
+              isChecked={changePassword}
+              onChange={() => setChangePassword(!changePassword)}
+            />
+          )}
+          {changePassword && usingHashedPassword && (
+            <Content isEditorial>
+              {_("Using a hashed password.")}{" "}
```

Comment:
Would be nice if someone finds a better wording for this.

Reply:
I guess we could re-evaluate the wording of the whole section. But I don't feel inspired today so I suggest to merge as it is (specially since we need to build an image with all the functional changes ASAP) and think about it later.